README.md

```
@@ -1 +1,197 @@
-add file ".gitignore"
+# Veri-Q 서버 접속 및 개발 가이드
+ 구축한 GCP 서버 인프라를 사용하기 위한 가이드입니다.
+
+## 🌐 서버 정보
+- **공통 외부 IP**: `34.64.218.236`
+- **관리자 계정**: `root` (DB 접속 시 사용)
+* **서버 사용자**: `sk38808738` (SSH 접속 시 사용)
+
+### 서비스별 포트
+| 서비스 | 포트 | 용도 |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)